ganttproject/src/net/sourceforge/ganttproject/search/SearchUiImpl.java

```diff
@@ -122,8 +123,11 @@ public void accept(final List<SearchResult<?>> results) {
       scrollPane.setBorder(BorderFactory.createEmptyBorder());
       final JPopupMenu popup = new JPopupMenu();
       popup.add(scrollPane);
+      popup.setPopupSize(searchBox.getWidth(), 300);
```

Comment:
Not sure how I feel about this, but being dynamic with the height seemed a bit beyond the scope of the changes.